app/assets/javascripts/bastion/components/bst-edit.directive.js

```diff
@@ -290,13 +290,13 @@ angular.module('Bastion.components')
 
         getIds = function (models) {
             models = models || [];
-            return _.pluck(models, "id");
+            return _.map(models, "id");
```

Comment:
I do not think it is the same, `_.property` returns a function, [jsbin](https://jsbin.com/difixo/2/edit?html,console) seems to agree...